### ISSUE TITLE
[Suggestion]: Dasboard Row Alignment

### ISSUE THREAD

**Issue by RossLloyd** [reef-defi/reef-explorer#5]
ammend CSS for row alignment:

![image](https://user-images.githubusercontent.com/25029366/113862248-12266600-97a0-11eb-8413-84c24dc2712a.png)

change 
@media (min-width: 992px)
.d-lg-block {
    display: block!important;
}

to

@media (min-width: 992px)
.d-lg-block {
    display: table-cell!important;
}

change
@media (min-width: 1200px)
.d-xl-block {
    display: block!important;
}

to

@media (min-width: 1200px)
.d-xl-block {
    display: table-cell!important;
}

etc.....

basically: display: table-cell is correct instead of display: block.

alignment then matches after:

![image](https://user-images.githubusercontent.com/25029366/113862626-8d881780-97a0-11eb-9ac5-ca0ce3e8c622.png)

**Comment by mariopino** [comment]
Thanks @RossLloyd, a redesign is currently being done so this is outdated.
